terraform/terraform.tfvars.example

```diff
@@ -0,0 +1,12 @@
+# Ansible Vault password file
+ansible_vault_id = "../ansible-vault/vault-pass"
```

Comment:
Is this just a plaintext password file ?

Reply:
It depends, could be plain text, but could also be a script. "The source can be a prompt, a file, or a script, depending on how you are storing your vault passwords." (https://docs.ansible.com/ansible/latest/user_guide/vault.html#managing-multiple-passwords-with-vault-ids)

Reply:
Okay, perfect. Could you maybe add this info/url as a comment?

Reply:
Sure, it's in https://github.com/adfinis-sygroup/vault-raft-backup-agent/pull/9